### ISSUE TITLE
Hashing

### ISSUE THREAD

**Issue by aaronpeikert** [aaronpeikert/reproducible-research#4]
@gabrielesh pointed me to an error in our approach on hashing, we should think it over.
What are your thoughts @brandmaier ?
maybe we should hash the file instead of the R Object?

**Comment by aaronpeikert** [comment]
https://github.com/aaronpeikert/workflow-showcase/issues/2

**Comment by brandmaier** [comment]
Yes, very good point!

**Comment by aaronpeikert** [comment]
So there is [md5sum-function](https://stat.ethz.ch/R-manual/R-devel/library/tools/html/md5sum.html) that comes with R to hash files, which would be a good fit for the workflow-example, but in the manuscript, we have no file.
On the other hand, I have realised that we just have failed to rebuild the docker images after fixing the date, so that an update of the digest package lead to another hash. So if we had done everything as we prescribed ourself, we would not have this problem.
